atlp-rwanda/falcons-ec-be
# #184637253 Two factor authentication for sellers

#### What does this PR do?

- adds two factor authentication for sellers

#### Description of Task to be completed?

Users that require a higher level of security (Sellers) should be able to turn on 2FA where authentication to the application also incudes a an extra factor i.e SMS or Authentications codes.

#### How should this be manually tested?

- on login you receive an email with an otp code
- use that top code to verify
- the top expires in 5 minutes

#### Any background context you want to provide?

- for backend development purposes the link sent to the user which has a token is not yet operational until there is a frontend site ready
- for now, the token and otp code is to be pasted in the body of the request(via postman or swaggerUI)

#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637253
#### Screenshots (if appropriate)
 none
#### Questions:
 none
